spec/docker/container/list_response.cr

```diff
@@ -0,0 +1,29 @@
+require "../../spec_helper"
+
+subject = Docker::Container::ListResponse.from_json ListResponseSample
+
+describe Docker::Container::ListResponse do
+  it "parses the sample data" do
+    subject.id.should eq "8dfafdbc3a40"
+    subject.names.should eq ["/boring_feynman"]
+    subject.image.should eq "ubuntu:latest"
+    subject.imageid.should eq "d74508fb6632491cea586a1fd7d748dfc5274cd6fdfedee309ecdcbc2bf5cb82"
+    subject.command.should eq "echo i"
+    subject.created.should eq 1367854155
+    subject.state.should eq "Exited"
+    subject.status.should eq "Exit 0"
+    subject.labels["com.example.vendor"].should eq "Acme"
```

Comment:
need to add a test that labels isn't nil, then do the tests on a non-nil version